airbyte-integrations/connectors/source-amazon-ads/integration_tests/configured_catalog_profiles.json

```diff
@@ -0,0 +1,14 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "profiles",
+        "json_schema": {},
```

Comment:
Since you refactored configured catalogs, please fill `json_schema` properties for each configured catalog with their actual schemas.

This change improves the overall quality of the connector, so it is better to finish it than revert.

Reply:
updated!